src/System.Device.Gpio/Interop/Unix/libgpiod/V1/SafeChipHandle.cs

```diff
@@ -19,6 +19,7 @@ public SafeChipHandle()
     protected override bool ReleaseHandle()
     {
         LibgpiodV1.gpiod_chip_close(handle);
+        handle = IntPtr.Zero;
```

Comment:
Where is this handle being set to something other than IntPtr.Zero?

Reply:
Rewritten to make this more clear. Using a handle as return type to a pinvoke call sets the value implicitly.